cosmic-core/systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -451,19 +451,19 @@ def process(self):
 
     def deletevpn(self, ip):
         logging.info("Removing VPN configuration for %s", ip)
-        CsHelper.execute("ipsec auto --down vpn-%s" % ip)
-        CsHelper.execute("ipsec auto --delete vpn-%s" % ip)
+        CsHelper.execute("ipsec down vpn-%s" % ip)
+        CsHelper.execute("ipsec down vpn-%s" % ip)
```

Comment:
Yes, and it seems this is as designed (according to ACS PR 872). Feels a bit lame, we could test it without once we see this works?

![screen shot 2017-03-09 at 21 21 13 pm](https://cloud.githubusercontent.com/assets/1630096/23769088/73d70d9c-050e-11e7-8f53-7b0d86c2452c.png)